EasyPost.Tests/ClientTest.cs

```diff
@@ -23,7 +23,7 @@ public void TestClient()
             // we specifically want to test the getters/setters
             Client client = new(FakeApikey, timeoutMilliseconds: 5000);
 
-            Assert.Equal(5000, client.ConnectTimeoutMillisecondsInUse);
+            Assert.Equal(5000, client.TimeoutMillisecondsInUse);
```

Comment:
question: can we just call it `TimeoutMilliseconds`? What's the `InUse` bit?

Reply:
I have addressed this.

I plan on a separate PR later that will clean up the configuration process for clients, so this will likely change again in that work.